src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1315,58 +1195,139 @@ struct SurfacesData : BaseGlobalStruct
+    Array1D<Real64> SurfOutWetBulbTempEMSOverrideValue;   // value to use for EMS override of outdoor air wetbulb temp (C)
+    Array1D<bool> SurfWindSpeedEMSOverrideOn;             //  if true, EMS is calling to override the surface's outdoor wind speed
+    Array1D<Real64> SurfWindSpeedEMSOverrideValue;        // value to use for EMS override of outdoor wind speed (m/s)
+    Array1D<bool> SurfViewFactorGroundEMSOverrideOn;      // if true, EMS is calling to override the surface's view factor to ground
+    Array1D<Real64> SurfViewFactorGroundEMSOverrideValue; // value to use for EMS override of surface's view factor to ground
+    Array1D<bool> SurfWindDirEMSOverrideOn;               // if true, EMS is calling to override the outside wind direction
+    Array1D<Real64> SurfWindDirEMSOverrideValue;          // value to use for EMS override of outside wind direction (deg)
+
+    // Surface Properties
+    Array1D<int> SurfDaylightingShelfInd;           // Pointer to daylighting shelf
+    Array1D<bool> SurfSchedExternalShadingFrac;     // true if the external shading is scheduled or calculated externally to be imported
+    Array1D<int> SurfExternalShadingSchInd;         // Schedule for a the external shading
+    Array1D<bool> SurfHasSurroundingSurfProperties; // true if surrounding surfaces properties are listed for an external surface
+    Array1D<int> SurfSurroundingSurfacesNum;        // Index of a surrounding surfaces list (defined in SurfaceProperties::SurroundingSurfaces)
+    Array1D<bool> SurfHasLinkedOutAirNode;          // true if an OutdoorAir::Node is linked to the surface
+    Array1D<int> SurfLinkedOutAirNode;              // Index of the an OutdoorAir:Node
+    Array1D<bool> SurfExtEcoRoof;                   // True if the top outside construction material is of type Eco Roof
+    Array1D<bool> SurfExtCavityPresent;             // true if there is an exterior vented cavity on surface
+    Array1D<int> SurfExtCavNum;                     // index for this surface in ExtVentedCavity structure (if any)
+    Array1D<bool> SurfIsPV;                         // true if this is a photovoltaic surface (dxf output)
+    Array1D<bool> SurfIsICS;                        // true if this is an ICS collector
+    Array1D<bool> SurfIsPool;                       // true if this is a pool
+    Array1D<int> SurfICSPtr;                        // Index to ICS collector
+    Array1D<bool> SurfIsRadSurfOrVentSlabOrPool;    // surface cannot be part of both a radiant surface & ventilated slab group
+
+    // Surface ConvCoeff Properties
+    Array1D<int> SurfTAirRef;                     // Flag for reference air temperature
+    Array1D<Real64> SurfIntConvCoeff;             // Interior Convection Coefficient pointer (different data structure) when being overridden
+    Array1D<Real64> SurfExtConvCoeff;             // Exterior Convection Coefficient pointer (different data structure) when being overridden
```

Comment:
These should be integers. That should reduce the eio diffs.  Without seeing the declarations, I had assumed `SurfIntConvCoeff` was the actual convection coefficient value.  Perhaps these should be renamed, e.g. `SurfIntConvCoeffIndex` ?